thirdparty/build-thirdparty.sh

```diff
@@ -514,14 +514,26 @@ build_rocksdb() {
     export CFLAGS=$OLD_FLAGS
 }
 
+# sasl
+build_sasl() {
+    check_if_source_exist $SASL_SOURCE
+    cd $TP_SOURCE_DIR/$SASL_SOURCE
+    OLD_FLAGS=$CFLAGS
+    unset CFLAGS
+    ./autogen.sh --prefix=$TP_INSTALL_DIR --enable-gssapi=no --enable-static=yes --enable-shared=no
```

Comment:
```suggestion
    CFLAGS= ./autogen.sh --prefix=$TP_INSTALL_DIR --enable-gssapi=no --enable-static=yes --enable-shared=no
```

can use `CFLAGS= ...` to force reset the env effectively just the specific line.

Reply:
> can use `CFLAGS= ...` to force reset the env effectively just the specific line.

updated